pkg/minikube/cluster/commands.go

```diff
@@ -82,10 +116,82 @@ func GetStartCommand(kubernetesConfig KubernetesConfig) (string, error) {
 	return buf.String(), nil
 }
 
+var logsCommand = fmt.Sprintf(`
+if which systemctl 2>&1 1>/dev/null; then
+  sudo journalctl -u localkube
+else
+  tail -n +1 %s %s
+fi
+`, constants.RemoteLocalKubeErrPath, constants.RemoteLocalKubeOutPath)
+
 var localkubeStatusCommand = fmt.Sprintf(`
-if ps $(cat %s) 2>&1 1>/dev/null; then
-  echo "Running"
+if which systemctl 2>&1 1>/dev/null; then
+  sudo systemctl is-active localkube 2>&1 1>/dev/null && echo "Running" || echo "Stopped"
 else
-  echo "Stopped"
+  if ps $(cat %s) 2>&1 1>/dev/null; then
+    echo "Running"
+  else
+    echo "Stopped"
+  fi
 fi
 `, constants.LocalkubePIDPath)
+
+var localkubeConfigTmpl = `[Unit]
+Description=Localkube
+Documentation=https://github.com/kubernetes/minikube/tree/master/pkg/localkube
+
+[Service]
+Type=notify
+Restart=always
+RestartSec=3
+
+ExecStart=/usr/local/bin/localkube {{.Flags}} \
+--generate-certs=false --logtostderr=true --enable-dns=false --node-ip={{.NodeIP}}
+
+ExecReload=/bin/kill -s HUP $MAINPID
+
+[Install]
+WantedBy=multi-user.target
+`
+
+func GetStartCommandSystemd(kubernetesConfig KubernetesConfig) (string, error) {
+
+	flagVals := make([]string, len(constants.LogFlags))
+	for _, logFlag := range constants.LogFlags {
```

Comment:
The flag parsing should be shared across both start commands as much as possible.  Maybe a command to generate `/usr/local/bin/localkube {{.Flags}} \
 +--generate-certs=false --logtostderr=true --enable-dns=false --node-ip={{.NodeIP}}`
and then use that in both templates.